data/tutorials/gs_00_up_and_running.md

```diff
@@ -238,29 +258,39 @@ We will now install everything we need to get a complete development environment
 - UTop, an improved REPL
 - `dune-release` to release code to `opam-repository`, the central package directory of opam.
 
+#### OCaml Platform Tools on Unix
+
 All these tools can be installed in your current switch (remember that opam groups installed packages in independent switches) using the following command:
 
 ```shell
-# Unix
 $ opam install dune merlin ocaml-lsp-server odoc ocamlformat utop dune-release
-
-# Windows (Diskuv OCaml has known bugs that make this harder than it needs to be)
-$ opam pin remove fiber omd stdune dyn ordering dot-merlin-reader yojson --no-action
-$ opam install dune merlin ocaml-lsp-server odoc ocamlformat utop
 ```
 
 Now that the tools are installed, it remains to understand how to use them. Most of them will be driven either by the editor or by Dune, but UTop is handy to try OCaml or a specific library.
 
-## Up and Running With the Platform Installer
+#### OCaml Platform Tools on Windows
+
+The DKML installer already installed all of the OCaml Platform tools, except for Merlin.
+
+If need to use Merlin, you can install it in an `opam` switch that you created using `dkml init`:
 
-As of 2023, the [OCaml Platform
-Installer](https://github.com/tarides/ocaml-platform-installer), presented
-here, is still experimental and in active development. It is an alternative way
-to install all the tools of the OCaml platform.
+```shell
+$ opam install merlin
+```
+
+## One-Step Installation on Unix: The Platform Installer
```

Comment:
This breaks the link `[See
below](#up-and-running-with-the-platform-installer)`